examples/evaluation_and_profiling/email_phishing_analyzer/src/nat_email_phishing_analyzer/register.py

```
@@ -65,7 +65,8 @@ async def _analyze_email_phishing(text: str) -> str:
 
         try:
             # Get response from LLM
-            response = await llm.apredict(config.prompt.replace("{body}", text))
+            response = await llm.ainvoke(config.prompt.replace("{body}", text))
+            response = str(response.content)
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)